examples/nlp/language_modeling/prepare_truncated_data.py

```diff
@@ -51,34 +60,44 @@ def _process_line(line, task, tokenizer, max_seq_length, tokens_to_generate_extr
         truncated_text = prompt.replace("{context}", truncated_text)
     return truncated_text
 
-def process_data(tokenizer: TokenizerSpec, prompt:str, task: str, max_seq_length: int, data_dir: str, n_jobs: int = -1):
+
+def process_data(
+    tokenizer: TokenizerSpec, prompt: str, task: str, max_seq_length: int, data_dir: str, n_jobs: int = -1
+):
     task = task.lower()
     if task not in TASKS:
         raise NotImplementedError(f"{task} not implemented")
-    
+
     task_file = f"{data_dir}/{TASKS[task]['subset']}"
     if not os.path.exists(data_dir) or not os.path.exists(task_file):
         raise ValueError(f"{data_dir} or {task_file} not found")
-    
+
     tokens_to_generate = TASKS[task]["tokens_to_generate"]
     prompt = None if prompt in ["None", None, ""] else prompt
     if prompt is not None:
         # when passing via command line, "\n" is changed to "\n\n"
         prompt = prompt.replace("\\n", "\n")
         assert "{prompt}" in prompt, "prompt must contain {prompt} for replacement"
         tokens_to_generate += len(tokenizer.text_to_tokens(prompt.replace("{prompt}", "")))
```

Comment:
What does this mean? Do we have a example how we get `tokens_to_generate` if we have a prompt_template?

Reply:
@ekmb